aws/resource_aws_mq_broker_test.go

```diff
@@ -644,7 +644,7 @@ func TestAccAWSMqBroker_updateTags(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_mq_broker.test", "tags.env", "test"),
 				),
 			},
-			// Adding new user + modify existing
```

Comment:
These comments were inaccurate - probably the testing functions were copied from `TestAccAWSMqBroker_updateUsers`